simapp/app.go

```diff
@@ -72,11 +72,6 @@ var (
 		staking.NotBondedPoolName: {supply.Burner, supply.Staking},
 		gov.ModuleName:            {supply.Burner},
 	}
-
-	// module accounts that are allowed to receive tokens
-	allowedReceivingModAcc = map[string]bool{
-		distr.ModuleName: true,
```

Comment:
so transferring to the community pool is disabled now?

Reply:
No, users can still send a `MsgFundCommunityPool` message.